ticdc/troubleshoot-ticdc.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Common Problems and Troubleshooting
+summary: Fix common problems while using TiCDC.
+category: reference
+aliases: ['/docs/dev/reference/tools/ticdc/troubleshoot-ticdc/']
+---
+
+# Common Problems and Troubleshooting
+
+This document includes some common problems while using TiCDC and appropriate solutions. It also summarizes some common operation failures and ways to fix them.
+
+## How to choose start-ts while starting a task
+
+First, you should know that the `start-ts` of a replication task corresponds to a TSO of the upstream TiDB cluster. The replication task will start its data request from this TSO. Therefore, the `start-ts` should meet two conditions below:
```

Comment:
Limit the use of `should` in user documentation.

Reply:
I found that `should` (111 times), `need(s) to` (125 times) and `must` (118 times) are almost equally used in current TiDB docs and their links, but `should` seldom goes with `user` or `you` . So, I changed some of `should` to `need to` or `must`.